lib/devcycle-ruby-server-sdk/api/devcycle_api.rb

```diff
@@ -182,9 +182,7 @@ def variable(user_data, key, default, opts = {})
       if local_bucketing_initialized? && @local_bucketing.has_config
         type_code = variable_type_code_from_type(type)
         variable_pb = variable_for_user_pb(user_data, key, type_code)
-        if variable_pb == nil
-          @logger.warn("No variable found or type mismatch for key #{key}, returning default value")
-        else
+        unless variable_pb.nil?
```

Comment:
should we just downgrade it to `debug` instead?

Reply:
i think we should remove it for now. We shouldn't actually be logging for the not found case like Adam said